### ISSUE TITLE
Score Viewer- view of score of their favorite match

### ISSUE THREAD

**Issue by nik132-eng** [nik132-eng/chrome-extensions#20]
@shivanshsin0203 give an update on this issue & please give the program name for contributing.

**Comment by shivanshsin0203** [comment]
I am working on this extension and I will raise an issue within 2-3 days regarding a feature on which I am working. I am contributing to KWOC 2023

**Comment by shivanshsin0203** [comment]
Hey, @nik132-eng  I have made the initial setup for the extension created a manifest file, and added a logo with a popup. How should I proceed? Should I create a new issue or a new PR by committing changes?

**Comment by nik132-eng** [comment]
New PR by commiting changes

**Comment by nik132-eng** [comment]
#25

**Comment by nik132-eng** [comment]
any further update @shivanshsin0203?

**Comment by shivanshsin0203** [comment]
@nik132-eng  Yes I will be adding  a new PR related to a new feature by tomorrow
